server/src/infra/graphql/query.rs

```diff
@@ -530,9 +547,48 @@ impl<Handler: BackendHandler> AttributeList<Handler> {
             .collect()
     }
 
-    fn extra_ldap_object_classes(&self) -> Vec<String> {
+    fn extra_object_classes(&self) -> Vec<String> {
         self.extra_classes.iter().map(|c| c.to_string()).collect()
     }
+
+    fn hardcoded_object_classes(&self) -> Vec<String> {
+        vec![
+            "inetOrgPerson".to_string(),
+            "posixAccount".to_string(),
+            "mailAccount".to_string(),
+            "person".to_string(),
+        ]
+    }
+
+    fn object_classes(&self) -> Vec<ObjectClassGQL> {
+        let mut all_object_classes: Vec<ObjectClassGQL> = vec![
+            ObjectClassGQL {
+                object_class: "inetOrgPerson".to_string(),
```

Comment:
These should probably just be generated based on the vector defined above (or the shared definition).